src/views/ajax.js

```diff
@@ -0,0 +1,76 @@
+const util = require('../lib/util');
+const dom = require('../lib/dom');
+
+let count = 0;
+let ready = false;
+let preRenderCache = [];
+let currentTimout = undefined;
+
+function update(method, uri, duration, size, status, statusText, time, contentType, requestId) {
```

Comment:
Can we shrink this down to one argument, `function update(config = {method, uri, ...})`? Would be much more clear.